### DISCUSSION TITLE
Inter v4 beta 1

### DISCUSSION THREAD

**Discussion by rsms** [rsms/inter#493] [Announcements]
[releases/v4.0-beta1](https://github.com/rsms/inter/releases/tag/v4.0-beta1)

This is a beta release of the upcoming version 4.0.
This is a major release with considerable changes compared to version 3.

Highlights:
- Six new additional "Display" designs which used to be a separate font, Inter Display. Now integrated into the variable font as an optical-size axis (`opsz`)
- Coordinate system changed to 2048. This addresses some issues with older (10+ years) postscript printer drivers and some software
- slnt (italic) variable-font axis has been removed. This addresses issues with web browsers unable to reliably apply italic styles. Also makes it possible for the Italic designs to diverge from the upright "roman" design, which is not possible when italic is an axis of one variable font.
- Lots of improvements to glyph design of all styles

---

See also: [In depth beta testers' discussion](https://github.com/rsms/inter/discussions/463)

**Comment by iduuck** [comment]
Awesome! Would be nice to get a comparison of the "standard" inter font, v3 vs. v4.

**Reply by iduuck** [reply]
Thanks @hydino2085143, but I mean, where are the key differences, like why should you use the one over the other. Something like a guide, why and where to use the Display font, and what it makes so special for display usage.

**Reply by rsms** [reply]
That’s a good idea @iduuck — there are thousands of little differences as many of the regular “text” styles have seen updates too, in addition to the six new “display” designs. There are also kerning changes, changes to opentype features, removal of slnt VF axis and much more. As usual, the release post will include a list. For now I recommend testing Inter v4 in as many scenarios as possible. Comparing it with v3 probably won’t be helpful beyond curiosity.

**Comment by nestflow** [comment]
Nice to see Inter Display comes into beta! What's the difference of Inter and Inter-V fonts in variable folder of release zip? And I guess `opsz=32` makes variable font be like Inter Display?

**Reply by rsms** [reply]
The only difference is the family name. Allows you to install this version alongside the stable one (or with static fonts, as many OSes still done any have great VF support.) The zip contains this document: https://github.com/rsms/inter/blob/master/misc/dist/variable.txt

opsz=32 indeed causes the six new “display” designs to be used (subject to wght and italic config.)

**Comment by boldtext** [comment]
On my PC, the optical size only goes up to 28 in Illustrator. Is this correct? I see that there are still ink wells. On my Mac, the optical size goes up to 32, but I don't know if it's the most recent version.
